pkg/abstract/map-map.go

```diff
@@ -46,6 +46,13 @@ func (wrapper MapValueMapper[Key, Val, Mapped]) Get(key Key) (Mapped, bool) {
 	return wrapper.mapVal(val), true
 }
 
+func (wrapper MapValueMapper[Key, Val, Mapped]) ContGet(key Key, cont func(Mapped)) {
```

Comment:
The name is not clear (to me). Missing documentation on exported (non-obvious, to me) function.

Reply:
This is an implementation of a method of the `MapToLocked` interface, which has an extensive comment explaining this method and its name. This method is a continuation-passing-style "get".